util/DataManager.js

```diff
@@ -0,0 +1,11 @@
+export default {
+    UpdateActivityList: (newList) => {
```

Comment:
mantenha sempre o padrão camelCase :D

```suggestion
    updateActivityList: (newList) => {
```